tests/experimental/datasources/test_config.py

```diff
@@ -204,7 +284,13 @@ def test_catch_bad_top_level_config(
                     "column_name": "foo",
                 },
             },
-            ("xdatasources", "assets", "column_splitter", "method_name"),
+            (
+                "xdatasources",
+                "assets",
+                "unknown splitter",
+                "column_splitter",
+                "method_name",
+            ),
```

Comment:
These more specific location errors are because of the change to `_load_asset_subtype` which results in asset errors being raised by the asset validators rather than the datasource validators.